activities/models/fan_out.py

```diff
@@ -242,9 +242,33 @@ def handle_new(cls, instance: "FanOut"):
                 except httpx.RequestError:
                     return
 
+            # Handle move for local follower
+            case (FanOut.Types.identity_moved, True):
+                from users.services import IdentityService
+
+                identity = instance.subject_identity
+                if identity.has_moved() and identity.aliases:
+                    follower = IdentityService(instance.identity)
+                    new_identity = Identity.by_actor_uri(identity.aliases[0])
+                    follower.unfollow(identity)
+                    follower.follow(new_identity)
+                    # TODO send MOVE notification to follower
```

Comment:
Is there a move notification type? Might just be an unfollow/follow - any idea what Mastodon does here?

Reply:
Interesting - probably worth checking out client compat with some of these. I suspect there's not much harm in sending an unsupported type. But not going to hold this up for it.